packages/vite-plugin-svelte-docgen/src/transform.js

```diff
@@ -44,8 +44,7 @@ export function transform_encoded(ast) {
 								type: "Identifier",
 								name: "Map",
 							},
-							// NOTE: Visit nested nodes _(if exists)_ to look for properties needed for further transformation
-							arguments: [ctx.visit(node.value)],
+							arguments: [node.value],
```

Comment:
Currently, `["events", "exports", "props", "types", "slots"]` are all top-level properties, so we can skip this traversal.